src/lti/outcome_response.py

```diff
@@ -163,4 +163,4 @@ def generate_response_xml(self):
             text_string = etree.SubElement(result_score, 'textString')
             text_string.text = str(self.score)
 
-        return '<?xml version="1.0" encoding="UTF-8"?>' + etree.tostring(root)
+        return etree.tostring(root, xml_declaration=True)
```

Comment:
I _think_ this should be approximately the same, but it's changing tests, so technically breaking backward compatibility. I'd prefer at this point to just make the prefix a bytestring (i.e. `b'<?xml'`), so that the tests can be exactly the same in Python 2. This is probably an improvement to the code, but I'd like to leave it for future work, rather than the Python 3 compatibility.